AutomatticTracks/build.gradle

```diff
@@ -39,6 +41,7 @@ android {
 
     lintOptions {
         baseline file("lint-baseline.xml")
+        warningsAsErrors true
```

Comment:
This helps in lint warning awareness. Lint warnings are super easy to overlook (there's usually just one line on CI that looks like this: `Ran lint on variant release: 2 issues found`). 

Marking warnings as errors shouldn't slower a development process as there's baseline file defined so it is still possible to pass a CI build with warnings.